src/Extensions/MemberExtension.php

```diff
@@ -40,8 +40,10 @@ public function updateCMSFields(FieldList $fields)
             'Root.Main',
             $sessionManagerField = SessionManagerField::create(
                 'SessionManagerField',
-                _t(__CLASS__ . '.SESSION_MANAGER_SETTINGS_FIELD_LABEL', 'Devices'),
-                $this->owner->ID
+                _t(__CLASS__ . '.DEVICES', 'Devices'),
+                $this->owner->ID,
+                _t(__CLASS__ . '.LEARN_MORE', 'Learn more'),
+                'http://userhelp.silverstripe.org/TODO'
```

Comment:
Link needs updating.

Reply:
No userhelp to link to right now =)  I've added it as an AC on the [userhelp issue](https://github.com/silverstripe/silverstripe-session-manager/issues/15)